templates/pr-create.md.ejs

```diff
@@ -1,5 +1,55 @@
-Thanks for the pull request, @<%= payload.sender.login %>! It looks like this is your first time contributing to ESLint, so please take a moment to read over our [contribution guidelines](http://eslint.org/docs/developer-guide/contributing/pull-requests).
+<%
 
-We'll also need you to sign our [CLA](http://eslint.org/cla), which is just a way for you to say that you give us permission to use your contribution.
+var FLAG_PATTERN = /^(Fix|Update|New|Breaking|Build|Docs|Upgrade):/,
+    ISSUE_REF_PATTERN = /\((fixes|refs) #\d+.*?$/;
 
-If you have any questions about the process, don't hesitate to ask.
+function isValidCommitFlag(log) {
+    var result = log.match(FLAG_PATTERN);
+    return !!result || log.indexOf("Revert \"") === 0;
+}
+
+function needsIssueRef(log) {
+    var result = log.match(ISSUE_REF_PATTERN);
+    return !result && log.indexOf("Docs:") !== 0;
+}
+
+var problems = [];
+
+// Check for one commit per pull request
+if (payload.commits > 1) {
+    problems.push("We require one commit per pull request. Please [squash](https://egghead.io/lessons/javascript-how-to-squash-multiple-git-commits) your commits.");
+} else if (meta.commits) {
+    var log = meta.commits[0].commit.message;
+    if (!isValidCommitFlag(log)) {
+        problems.push("The commit summary needs to begin with a tag (such as `Fix:` or `Update:`). Please check out our [guide](http://eslint.org/docs/developer-guide/contributing/pull-requests#step-2-make-your-changes) for how to properly format your commit summary.")
```

Comment:
Can we add some quick instruction on how to do amend to pull requests? That information is not really all that easy to find, and I think would be helpful. Or at least a link to http://eslint.org/docs/developer-guide/contributing/pull-requests#updating-the-commit-message